bbs/CMakeLists.txt

```diff
@@ -152,3 +152,4 @@ add_library(bbs_lib ${COMMON_SOURCES} ${ZMODEM_SOURCES} ${PLATFORM_SOURCES})
 target_link_libraries(bbs_lib localui core sdk ${CL342_LIB} ${CMAKE_THREAD_LIBS_INIT})
 add_executable(bbs ${BBS_MAIN})
 target_link_libraries(bbs bbs_lib)
+target_link_libraries(bbs ssh)
```

Comment:
Just make sure that the libssh-dev package (if using debian/ubuntu flavor) is installed to get header files in /usr/include/libssh (otherwise compile libssh from source and install).

Compiling wwiv presumes that libssh (both the .so library and header files) are present in the dev system in the path. (Not unlike it's assumptions for cryptlib)